spark-plugins/pom.xml

```diff
@@ -22,13 +22,11 @@
     <groupId>co.cask.hydrator</groupId>
     <version>1.5.0-SNAPSHOT</version>
   </parent>
-
   <properties>
     <app.parents>system:cdap-data-pipeline[4.0.0-SNAPSHOT,4.1.0-SNAPSHOT),system:cdap-data-streams[4.0.0-SNAPSHOT,4.1.0-SNAPSHOT)</app.parents>
-    <spark.version>1.3.1</spark.version>
+    <spark.version>1.6.2</spark.version>
```

Comment:
Kinesis spark streaming needs the newer version of spark streaming library, other spark streaming plugins dependent on this library are building and passing the unit tests.